debian/rules

```diff
@@ -85,7 +85,8 @@ override_dh_auto_configure:
 		--default-library=shared \
 		-Dper_library_versions=false \
 		-Dinclude_subdir_arch=../$(DEB_HOST_MULTIARCH)/dpdk \
-		-Dmachine=$(RTE_MACHINE)
+		-Ddisable_drivers=compress/isal,compress/mlx5,net/mlx4,net/mlx5,vdpa/mlx5,common/mlx5,regex/mlx5,raw/ioat \
+		-Dmachine=generic
```

Comment:
Why is this being changed from `default` (line 25) to `generic` here, and why is this change not being made on line 25 to preserve the functionality of the `RTE_MACHINE` override?

Reply:
default no longer seemed to work and generic matches what spdk builds  dpdk with. I would prefer to land like this and make the suggested changes regarding RTE_MACHINE in a subsequent PR so we can get https://github.com/daos-stack/daos/pull/9063 which is ready to land onto master so we can start working with updated SPDK. does that sound okay?

Reply:
We also need to land the DAOS-test-branch-*: *nobest* PRs first or he test phase of landing this will fail.

Reply:
@brianjmurrell great, I will repush with line 25 changes and update the changelog dates on this PR.
Is there anything that's stopping the nobest PRs from landing or can we get those landed before I repush?